docs/kafka/metrics-monitoring-kafka/README.adoc

```diff
@@ -127,6 +127,10 @@ The Kafka instance type determines the maximum connection creation rate.
 `kafka_topic:kafka_topic_partitions:sum`:: Number of partitions across all topics in the cluster. This metric does not include partitions from internal Kafka topics, such as `\__consumer_offsets` and `__transaction_state`.
 +
 The Kafka instance type determines the maximum number of partitions.
+
+`kas_broker_partition_log_size_bytes_top50`:: Sizes, in bytes, of the fifty largest topic partitions on each broker in the cluster. The total amount of storage being used by all topic partitions on a broker is shown by the `kafka_broker_quota_totalstorageusedbytes` topic metric. The total usage for a broker must stay below the `kafka_broker_quota_softlimitbytes` value to avoid throttling of producers. 
```

Comment:
Suggest putting the last sentence about avoiding throttling into a note for added emphasis.

Reply:
Yeah, I did consider that. But it's stated here twice with these two metrics. And then it's repeated again directly underneath, in the **Broker metrics** section. I think that's sufficient.